runtime/lua/vim/_meta/api_keysets_extra.lua

```diff
@@ -173,6 +173,26 @@ error('Cannot require a meta file')
 --- @field force? true
 --- @field cterm? vim.api.keyset.hl_info.cterm
 
+--- @class vim.api.keyset.get_keymap
+--- @field abbr? 0|1
```

Comment:
does there shouldn't boolean? and in doc says is return maparg() like dict maybe we can also link this object for maparg?

Reply:
No, they are integers

Reply:
I checked the result of `:lua =vim.api.nvim_get_keymap('n')` and found out that fields that should be boolean are represented by 0/1

Reply:
> maybe we can also link this object for maparg?

`maparg()` can returns the rhs of a mapping if `{dict}` if falsy or not specified, e.g. `maparg('<C-l>')`. Currently it is annotated to return a string.

We can do this for `maplist()` though which gives the same result as `nvim_get_keymap`